src/test/kotlin/org/rust/lang/core/type/RsStdlibExpressionTypeInferenceTest.kt

```diff
@@ -74,6 +74,15 @@ class RsStdlibExpressionTypeInferenceTest : RsTypificationTestBase() {
         }
     """)
 
+    fun `test RangeToInclusive new syntax`() = stubOnlyTypeInfer("""
+    //- main.rs
+        fn main() {
+            let x = ..=42u16;
+            x
```

Comment:
Let's add `;` to have correct code in tests
It doesn't affect type inference here but I think it's a good habit to write correct code in such tests to avoid weird results in some cases

Reply:
Thanks fixed, i also adjusted the code to make sure we do not reintroduce type errors in the future.